std_msgs/Int16[] mav_id
std_msgs/Float64[] depth
geometry_msgs/Vector3[] position

